src/custom_node.cpp

```diff
@@ -35,16 +36,7 @@ CustomNode::CustomNode(
     library(library),
     parameters(parameters),
     nodeOutputNameAlias(nodeOutputNameAlias) {
-    if (parameters.size() > 0) {
-        this->libraryParameters = std::make_unique<struct CustomNodeParam[]>(parameters.size());
-
-        int i = 0;
-        for (const auto& [key, value] : parameters) {
-            this->libraryParameters[i].key = key.c_str();
-            this->libraryParameters[i].value = value.c_str();
-            i++;
-        }
-    }
+    this->libraryParameters = createCustomNodeParamArray(this->parameters);
```

Comment:
Couldn't we initialize this field as well?

Reply:
Done